README.md

```diff
@@ -27,54 +26,52 @@
 **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, and synonyms are provided out-of-the-box.
 For more information about features go to [our documentation](https://docs.meilisearch.com/).
 
-<p align="center">
-  <a href="https://crates.meilisearch.com"><img src="assets/crates-io-demo.gif" alt="crates.io demo gif" /></a>
+<p align="center" id="demo">
```

Comment:
Why have you added an `id`?

Reply:
It was for the link that we suppressed, I just remove id